New paragraph on not requiring remote participation for all events (#16)
mirjak/draft-kuehlewind-shmoo-remote-fee: draft-ietf-shmoo-remote-fee.md

This document extends this principle to explicitly cover online
participation at meetings.

> However, this document does not automatically advise that all kind of IETF meetings and events

Reviewer: Maybe "This document does not recommend that all types of IETF meetings and events must have a remote participation option, as there could be ..."

Author: Actually I didn't want to use "recommend" because enabling remote participation whenever possible seems like a good recommendation. I was initially using "require" instead of "advise" but that seemed too strong...

Reviewer: Okay, I appreciate your concerns about advise/require/recommend.  I just think changing the sentence structure to "This doc doesn't ... because ..." is better.

Author: Not sure which change you are looking for now. Can you make another proposal?

Reviewer: "This document does not advise that all IETF meetings or events must have a remote participation item, because there could be technical or other reasons why that might not always be possible."

> However, this document does not automatically advise that all kind of IETF meetings and events
> must also have a remote participation option, as there could be technical or other reasons why that
> might not always be possible. This document rather says that if remote participation is provided,

Author: ```suggestion
This document does not advise that all IETF meetings or events must have a remote participation item,
because there could be technical or other reasons why that might not always be possible.
This document rather says that if remote participation is provided,
```
